profiles/gcc-10-aix-72

```diff
@@ -6,6 +6,9 @@ os.version=7.2
 compiler=gcc
 compiler.version=10.3
 compiler.libcxx=libstdc++
+compiler.threads=None
```

Comment:
We use `pthreads` for AIX - so unless that breaks something we'll want to use that